src/activity_handlers.cpp

```diff
@@ -2040,21 +2040,6 @@ void activity_handlers::aim_finish( player_activity *, player * )
     return;
 }
 
-void activity_handlers::washing_finish( player_activity *act, player *p )
-{
-    item &filthy_item = p->i_at( act->position );
-
-    if( p->is_worn( filthy_item ) ) {
```

Comment:
Did you test washing worn filthy clothing? Is it working as it should?

Reply:
Just tested it again, it works.  The item stays worn and is cleaned.

Reply:
Ok, thanks.